Gemfile

```diff
@@ -1,3 +1,5 @@
 source "https://rubygems.org"
 
+gem 'pharos-kube-client', :git => 'https://github.com/kontena/pharos-kube-client', :ref => 'cf73aa1'
```

Comment:
Consider making a release/gem

Reply:
I would also prefer to have a more generic name because it's not Pharos specific library. Now it has unnecessary "namespace"... just saying, not a blocker.

Reply:
Yes, I removed the `[WiP]` too soon, this versioning is still dev. The `pharos-kube-client` may still want some API review before making a release.